docs/7.migration/7.component-options.md

```diff
@@ -96,23 +96,23 @@ See [layout migration](/docs/migration/pages-and-layouts).
 
 ## `loading`
 
-This feature is not yet supported in Nuxt 3.
+This feature is not supported in Nuxt 3 yet.
 
 ## `middleware`
 
 See [middleware migration](/docs/migration/plugins-and-middleware).
 
 ## `scrollToTop`
 
-This feature is not yet supported in Nuxt 3. If you want to overwrite the default scroll behavior of `vue-router`, you can do so in `~/app/router.options.ts` (see [docs](/docs/guide/directory-structure/pages/#router-options)) for more info.
+This feature is not supported in Nuxt 3 yet. If you want to overwrite the default scroll behavior of `vue-router`, you can do so in `~/app/router.options.ts` (see [docs](/docs/guide/directory-structure/pages/#router-options)) for more info.
 
 ## `transition`
 
 See [layout migration](/docs/migration/pages-and-layouts).
 
 ## `validate`
 
-The validate hook in Nuxt 3 only accepts a single argument, the `route`. Just as in Nuxt 2, you can return a boolean value. If you return false and another match can't be found, this will mean a 404. You can also directly return an object with `statusCode`/`statusMessage` to respond immediately with an error (other matches will not be checked).
+The validate hook in Nuxt 3 only accepts a single argument, the `route`. Just as in Nuxt 2, you can return a boolean value. If you return false with no other match found, this will mean a 404. You can also directly return an object with `statusCode`/`statusMessage` to respond immediately with an error (Other matches will get ignored).
```

Comment:
You probably wanted to include a full stop.
```suggestion
The validate hook in Nuxt 3 only accepts a single argument, the `route`. Just as in Nuxt 2, you can return a boolean value. If you return false with no other match found, this will mean a 404. You can also directly return an object with `statusCode`/`statusMessage` to respond immediately with an error. (Other matches will get ignored.)
```